lib/msf/base/simple/noop_job_listener.rb

```diff
@@ -0,0 +1,44 @@
+require 'singleton'
+
+class NoopJobListener
+
+  include Singleton
+
+  def waiting(id)
+  end
+
+  def start(id)
+  end
+
+  def completed(id, result, mod)
+  end
+
+  def failed(id, error, mod)
+  end
+
+  def running?(id)
```

Comment:
Just noticed that we can get rid of the methods that aren't specific to the listener API :+1: